### PR TITLE
Collector auth tokens

### LINKED ISSUE #232: Add `GET /api/tasks/:task_id/collector_auth_tokens`

### PR DESCRIPTION
**Pull request by jbr**
closes #232

### CODE REVIEW

File: src/routes/tasks.rs [hunk 1]
```diff
@@ -93,3 +93,16 @@ pub async fn update(
...
 }
+
+pub mod collector_auth_tokens {
+    use super::*;
+    pub async fn index(
+        _: &mut Conn,
+        (task, client): (Task, AggregatorClient),
+    ) -> Result<impl Handler, Error> {
+        //        let leader = task.leader_aggregator(&db).await?;
+        //        let client = leader.client(client);
```
Comment (tgeoghegan) [inline src/routes/tasks.rs:104]: Commented out lines should be removed

**Review by tgeoghegan — APPROVED** [review]
This has the right shape, but it'll need to be rebased on #216 to account for minor API differences, right (e.g., some message fields being singular or non-optional).

**Comment by jbr** [comment]
Yeah whichever is merged second will need to resolve conflicts (if this one, the second commit in this PR can be dropped)

File: src/routes/tasks.rs (hunk 1 above)
Comment (jbr) [reply src/routes/tasks.rs:104]: In this case they're the lines we'll need post #216. I'll stash locally and remove

**Pull request merged**